lib/chained_job/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ChainedJob
-  VERSION = '0.6.1'
+  VERSION = '0.6.2'
```

Comment:
This is worth a minor version bump. So `0.7.0` is better.

Reply:
Yes that makes sense, bumped to 0.7.0